mk/config.mk

```diff
@@ -579,8 +579,10 @@ CFG_TA_MBEDTLS_SELF_TEST ?= y
 # It's also possible to configure to use mbedtls instead of tomcrypt.
 # Then the variables should be assigned as "CFG_CRYPTOLIB_NAME=mbedtls" and
 # "CFG_CRYPTOLIB_DIR=lib/libmbedtls" respectively.
-CFG_CRYPTOLIB_NAME ?= tomcrypt
-CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
+# if you want to use tomcrypt, use " CFG_CRYPTOLIB_NAME ?= tomcrypt" and
+# "CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt"
+CFG_CRYPTOLIB_NAME ?= mbedtls
```

Comment:
We usually change platform related conf.mk, not this global file. Please add them to conf.mk.